### PR TITLE
Edits to Deploying an application using odo Section of README

### PR DESCRIPTION
**Pull request by danielhelfand**
## What is the purpose of this change? What does it change?
Correcting typos in README on using `crc`/`minishift`

## Was the change discussed in an issue?
N/A

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -91,16 +91,16 @@ https://asciinema.org/a/225717[image:https://asciinema.org/a/225717.svg[asciicas
...
-More information on how to install and use `Minishift` can be found link:https://docs.okd.io/latest/minishift/getting-started/index.html[here], for `CodeReady Containers` it can be found link:https://code-ready.github.io/crc/[here].
+More information on how to install and use `Minishift` can be found link:https://docs.okd.io/latest/minishift/getting-started/index.html[here]. For `CodeReady Containers`, information can be found link:https://code-ready.github.io/crc/[here].
 
 . Start a local OpenShift development cluster by using `minishift` or `crc`.
 +
 [source,bash]
 ----
 $ minishift start
-$ # or
+# or
```
Comment (danielhelfand) [inline README.adoc]: Removed `$` since this is a comment, not a command executing.

**Comment by mohammedzee1000** [comment]
@cdrage make sure you do ok-to-test on prs that need them or prow will not run

**Pull request merged**